deploy/buildtools/buildconfig.py

```diff
@@ -65,9 +70,21 @@ def __init__(self,
         self.PLATFORM_CONFIG = recipe_config_dict['PLATFORM_CONFIG']
         self.post_build_hook = recipe_config_dict['post_build_hook']
 
+        bit_builder_recipe_file = recipe_config_dict['bit_builder_recipe']
+        schema_file = f"schemas/{bit_builder_recipe_file}"
+
+        def validate_helper(val_cond: bool):
+            if os.path.exists(schema_file):
+                if not val_cond:
+                    raise Exception(f"Invalid YAML in build recipe: {name}")
+            else:
+                rootLogger.warning(f"Unable to find schema file for {bit_builder_recipe_file}. Skipping validation.")
```

Comment:
Why not an error?

Reply:
If someone creates a new YAML file (instead of using the overrides) this currently gives a warning since the schema file path is derived from the path to the YAML file (i.e. `bit-builder-recipes/aws.yaml` -> `schemas/bit-builder-recipes/aws.yaml`).